src/Windows/dcap_provider_tests/dcap_provider_tests.vcxproj

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Because dcap_provider_ut is a standalone exe which exercises the dcap provider - it's not actually a unit test, it's an integration test. This is a unit test - it's linked with the source of the code and exercises components that are not exposed outside of the DLL.